### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by haadcode**
Added Orbit to the list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ place to ask about it might be in [ipfs/apps](https://github.com/ipfs/apps) or
 * [ipfs.pics](https://ipfs.pics) - Upload and share pics. [GitHub](https://github.com/ipfspics/server)
 * [IPFSBin](http://ipfsbin.xyz/) - Pastebin clone build. [GitHub](https://github.com/victorbjelkholm/ipfsbin)
 * [markup.rocks](https://github.com/davidar/markup.rocks) - Pandoc-based markup editor/previewer/converter, ported to IPFS. [Example](https://ipfs.io/ipfs/QmWPgJnUGLB1LPh9KMG9LEN4LVu5e17TwkEtcmTWdNn9V6/#/ipfs/QmfQ75DjAxYzxMP2hdm6o4wFwZS5t7uorEZ2pX9AKXEg2u)
+* [Orbit](https://github.com/haadcode/orbit) - Distributed, peer-to-peer chat application on IPFS-
```
Comment (RichardLitt) [inline README.md]: Remove dash at the end?

**Comment by RichardLitt** [comment]
Hells yeah! One small fix that is needed, then gtg.

**Comment by haadcode** [comment]
Fixed.

**Comment by RichardLitt** [comment]
<3!

**Pull request merged**